config.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'you-will-never-guess'
+    RPC_SOCKET = os.environ.get('RPC_SOCKET') or 'you-will-never-guess'
+    CONNECT_INFO = os.environ.get('CONNECT_INFO') or 'you-will-never-guess'
```

Comment:
The app should probably fail if these aren't set. It just won't work. So let's not set defaults here.

Reply:
It does fail, the `or` syntax doesn't work. Defaults have to be given to `get` as a second argument.